robot_sort/robot_sort.py

```diff
@@ -97,16 +99,91 @@ def sort(self):
         Sort the robot's list.
         """
         # Fill this out
-        pass
+        # Robot starts at position 0
+        #   Pick up current card and turn on light
+        #       true/false is used to indicate whether to continue a loop
+        #       move right and compare each card
+        #   compare > = 1, < = -1, 0 = 0, None
+        #   while light is on,
+
+        # THIS IS BUBBLE SORT: SET THE LIGHT TO TRUE,
+        #   CREATE WHILE LOOP THAT TELLS THE FUNCTION IT'S TIME TO SWAP (THE LIGHT)
+        #   SET LIGHT OFF AND CREATE A LOOP FOR MOVING RIGHT (WHILE CAN_MOVE_RIGHT)
+        #   IF THE COMPARED ITEM IS 1: SWAP EM, MOVE RIGHT, AND MAKE THE LIGHT TRUE AGAIN, THIS WILL RESTART THE NEXT ITERATION.
+        #   THE ROBOT REPRESENTS THE I OF THE FOR/WHILE LOOP, IT WILL TAKE IN THE INDEX
+
+        # ATTEMPT WITH LIGHT -------> CANNOT FIGURE THIS OUT FOR THE LIFE OF ME
+        # Turn light on AKA TRUE
+        # self.set_light_on()
+        # # While the light is true run through the loop
+        # while self.light_is_on():
+        #     # Swap item to get it in robots hands
+        #     self.swap_item()
+        #     # Move to the next available number
+        #     # self.move_right()
+        #     # # Immediately set light to off to control the loops
+        #     self.set_light_off()
+        #     # Create loop that is able to move right, and also not have empty spaces in the array
+        #     while self.can_move_right():
+        #         # if number being held is greater than the next, swap it, move right, and restart the loop (true)
+        #         self.move_right()
```

Comment:
I love how you tried two different sorts, there is a way to do this with bubble sort and selection sort so great job in understand of UPER!!